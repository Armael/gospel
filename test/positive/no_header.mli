val f : int -> unit

(* EXPECTED
   [0] OK
*)
