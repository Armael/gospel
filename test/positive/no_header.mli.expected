
*******************************
********** Parsed file ********
*******************************
val f : int -> unit

*******************************
****** GOSPEL translation *****
*******************************
(*@ open Stdlib *)

(*@ open Gospelstdlib *)

val f : int -> unit


*******************************
********* Typed GOSPEL ********
*******************************
module no_header.mli

  Namespace: no_header.mli
    Type symbols
      
    Logic Symbols
      
    Field Symbols
      
    Exception Symbols
      
    Namespaces
      
    Type Namespaces
      
  Signatures
    (*@ open Stdlib *)
    
    (*@ open Gospelstdlib *)
    
    val f : int -> unit
    

OK
